compose-files/staff/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.7"
+
+networks:
+  placeos_placeos:
+    external: true
+
+volumes:
+  postgres-data:
+
+services:
+
+  postgres:  # Database used by Staff API
+    image: postgres:${POSTGRES_VERSION:-13-alpine}
+    container_name: postgres
+    restart: unless-stopped
+    networks:
+      placeos:
+        ipv4_address: 172.31.231.17
```

Comment:
Is this needed? Looks like explicit IP allocation has been dropped in https://github.com/place-labs/partner-environment/pull/24/commits/1a40c64b6e09111a8ef89aec26206a9279acc09c

Reply:
yep agreed should work fine without it.